### ISSUE TITLE
vmctl influx: problem when tag and field have the same name

### ISSUE THREAD

**Issue by gmwiku** [VictoriaMetrics/VictoriaMetrics#1299]
**Describe the bug**
Generally it is not the briliant idea to have tag and field key with the same name but sometimes it happens:
https://docs.influxdata.com/influxdb/v1.8/troubleshooting/frequently-asked-questions/#tag-and-field-key-with-the-same-name
and that applies to my situation.
Unfortunatelly vmctl seems to not support such cases - I can see the queries on influxdb console, but data are not passed to victoriametrics and I'm even not sure if queries to influxdb returns any data.

**To Reproduce**
The problem occured during import big amount of data (import is still in progress), but I will try to prepare step by step instruction with subset of data.

**Version**
I use latest vmctl docker version:
victoriametrics/vmctl              latest       b5333d375fb4   13 days ago     18.5MB

**Comment by gmwiku** [comment]
**To Reproduce**

```
#! /bin/bash

docker run -d --rm --name influxdb -p "8086:8086" influxdb:1.8-alpine
docker run -d --rm --name victoriametrics -p "8428:8428" victoriametrics/victoria-metrics

curl -i -XPOST http://localhost:8086/query --data-urlencode "q=CREATE DATABASE mydb"
curl -i -XPOST 'http://localhost:8086/write?db=mydb' --data-binary 'meausrement,samename=tagvalue anothername=1.0'
sleep 1
curl -i -XPOST 'http://localhost:8086/write?db=mydb' --data-binary 'meausrement,samename=tagvalue samename=2.0'
sleep 1
curl -i -XPOST 'http://localhost:8086/write?db=mydb' --data-binary 'meausrement,samename=tagvalue anothername=3.0'

docker run --rm -i --name vmctl --add-host="influxdb:host-gateway" --add-host="victoriametrics:host-gateway" victoriametrics/vmctl influx --influx-addr http://influxdb:8086 --influx-database mydb --vm-addr http://victoriametrics:8428

sleep 10
curl -s http://localhost:8428/api/v1/export -d 'match[]={__name__!=""}'

docker rm -f influxdb victoriametrics
```
There should be at least 2 (ideally 3) samples imported.

**Comment by hagen1778** [comment]
Hi! Thanks for report! 
I see the problem. Looks like it can be solved if vmctl will explicitly add `::tag` suffix for filters:
```
select "anothername" from "meausrement" where "samename"::tag='tagvalue'
name: meausrement
time                anothername
----                -----------
1621318157534891000 1
1621318174074819000 3
```

It seems easy to fix, though. Are you aware of any complications of applying `::tag` suffix to queries? Could it affect performance?

**Comment by gmwiku** [comment]
Didn't notice any complications neither changes in Wiery performance.

**Comment by valyala** [comment]
FYI, `vmctl` should properly migrate Influx data with identical names for field and tag starting from [v1.60.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed.

@gmwiku , please re-open the issue if it isn't fixed in vmctl v1.60.0.
